### DISCUSSION TITLE
Zig zag around my mesh

### DISCUSSION THREAD

**Discussion by bkeys** [openscenegraph/OpenSceneGraph#1267] [Q&A]
So my code has been doing this for some time, I am not doing anything fancy with the scene graph and would be able to replicate this with pretty simple code. Do you have any idea why my edges are all zig zagged and how to stop this from happening? It only happens occasionally.

![the issue](https://i.imgur.com/N7JedEt.png)

After some playing with it, it seems to happen the most when something is pretty close to the camera

**Comment by robertosfield** [comment]
You don't provide any details about your scene graph, what rendering techniques, shaders tc are used, hardware etc. so there really isn't much we can say, it could be so many things.
